README.md

```diff
@@ -1,6 +1,6 @@
 # Sentry Github Actions App
 
-**NOTE**: You can try this project but you will need to deploy your own ingestion app. If this is a project you would like us to invest in, please let us know in [this issue](https://github.com/getsentry/sentry-github-actions-app/issues/46).
+**NOTE**: If this is a project you would like us to invest in, please let us know in [this issue](https://github.com/getsentry/sentry-github-actions-app/issues/46).
```

Comment:
You can ignore the doc changes if you like.

Overall, what I do is move information wrt to the deployment to `setup.md` and remove info about customers having to deploy the backend in order to test this.